sample-compose/app/src/main/res/navigation/nav_graph_main.xml

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<navigation xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/main_nav_graph"
-    app:startDestination="@id/homeFragment">
-
-    <fragment
-        android:id="@+id/homeFragment"
-        android:name="co.nimblehq.sample.compose.ui.screens.xml.HomeFragment"
-        tools:layout="@layout/fragment_home" />
-
-    <fragment
-        android:id="@+id/homeComposeFragment"
-        android:name="co.nimblehq.sample.compose.ui.screens.compose.HomeComposeFragment" />
-
-</navigation>
```

Comment:
Sooo compose doesn't make use of navigation component? 😯

Reply:
@Tuubz [it does](https://developer.android.com/jetpack/compose/navigation), but it goes through Kotlin instead of XML 😉

Reply:
Cool, but also quite unexpected/confusing, since we weren't using this before 💭 

Just to make sure, we can still combine `fragment navigation with xml` with `compose`, right?

I'm thinking if we should stick with the old approach first, so it's aligned with the `template`. It could be quite confusing when a developer compares `template` with `sample-compose`  🤔 

cc @nimblehq/android-chapter

Reply:
Resolved: https://github.com/nimblehq/android-templates/discussions/279